### ISSUE TITLE
animals is broken unless api key is set

### ISSUE THREAD

**Issue by SumWum** [Taaku18/modmail-plugins#7]
TL;DR random.cat's API returns a 503 for some reason, and unless an API key is set for thecatapi.com, the command just flat out breaks. Created a fork to fix this by just flat out disabling the default meow command like so:

`# async with self.bot.session.get("http://aws.random.cat/meow") as r:`
`    # cat = (await r.json())["file"]`
`embed = discord.Embed(title=":x: API Broken")`
`embed.description("The default cat API is down. In the meantime, use `{prefix}help meow` for more info.")`
`#embed.set_image(url=cat)`

Can be viewed in my fork [here](https://github.com/SumWum/modmail-plugins/commit/f1a166f503f019e32dc6fca3eea83499ed3e434f?diff=unified).
